mir-module/src/main/resources/xsl/response-mir-utils.xsl

```diff
@@ -152,12 +152,12 @@
           </xsl:when>
           <xsl:otherwise>
             <xsl:attribute name="class">page-item disabled</xsl:attribute>
-            <a class="page-link"><span>
```

Comment:
Hm, don't now why there is a span.
But I also can't imagine why you like to remove it.